.storybook/recipes/CalendarCard/CalendarCard.module.css

```diff
@@ -0,0 +1,59 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #CALENDAR CARD
+\*------------------------------------*/
+
+/**
+ * Card recipe for displaying calendar data
+ */
+.calendar-card {
+  width: 16rem;
+  justify-content: space-between;
+  gap: var(--eds-size-1);
+  background-color: var(--eds-theme-color-background-neutral-default-hover);
+}
+
+.calendar-card--brand {
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-brand-primary-strong);
+}
+
+.calendar-card--revise {
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-grade-revise-strong);
+}
+
+.calendar-card--success {
+  border-left: var(--eds-border-width-lg) solid
+    var(--eds-theme-color-border-utility-success-strong);
+}
+
+.calendar-card__title {
+  @mixin eds-typography-preset-007-bold;
+  display: flex;
+  align-items: center;
```

Comment:
I'm wondering if this should be `flex-start`, like in `calendar-card__dates`, so the icon always floats to the top when the text wraps.
<img width="267" alt="long text strings calendar card story in storybook" src="https://user-images.githubusercontent.com/7761701/173698137-af6634b0-5d19-4476-807a-700fac9966e3.png">

Reply:
done

Reply:
Thank you!